tripal_analysis_go.info

```diff
@@ -7,3 +7,4 @@ package = Tripal Extensions
 
 dependencies[] = tripal
 dependencies[] = tripal_chado
+dependencies[] = tripal_cv
```

Comment:
i actually think we want to remove all dependencies for v3 compatibility.  You don't want to be enabling legacy modules if you dont need them.

  Then in the install file, implement hook_requirements (see: tripal_analysis_blast)

https://github.com/tripal/tripal_analysis_blast/blob/6b622eb62e6d80b64b135bc7d31030de6e1c37f5/tripal_analysis_blast.install#L231-L248

Reply:
Do you mean removing tripal_cv from this file, and adding a check for it in hook_requirement?
I'm guessing leaving it in the .info file install it automatically, whereas the hook_requirement just send an alert if it is not installed?